packages/now-client/src/upload.ts

```diff
@@ -71,7 +71,7 @@ export async function* upload(
 
   const shas = missingFiles;
 
-  yield { type: 'file_count', payload: { total: files, missing: shas } };
+  yield { type: 'file-count', payload: { total: files, missing: shas } };
```

Comment:
You'll also need to update `packages/now-cli/src/util/deploy/process-deployment.ts` to reflect that change.